NEWS.md

```diff
@@ -6,11 +6,13 @@
 `AVALCATx` & `AVALCAxN`. (#2480)
 - New function `derive_vars_crit_flag()` for deriving criterion flag variables
 (`CRITy`, `CRITyFL`, `CRITyFLN`). (#2468)
-
 - Replace use of `data("sdtm")` with `sdtm <- pharmaverse::sdtm` in templates and vignettes. (#2498)
 
 ## Updates of Existing Functions
 
+- `compute_bsa(height, weight, method` - now uses improved version of DuBois-DuBois forumula for bsa (#2532)
```

Comment:
The wording could confuse users because they may assume that the results are different now.

Either we should state that the results are still the same or we shouldn't add a NEWS entry because the change is not user-facing.

@bms63 , what do you think?

Reply:
I think if no change for the user then lets suppress it from user-facing notes.  

@jimrothstein lets move it to developer notes just so we remember when we updated - for posterity!